### ISSUE TITLE
Support for custom data layer name

### ISSUE THREAD

**Issue by axel-havukangas-tt** [PiwikPRO/tracking-base-library#16]
Moved issue https://github.com/PiwikPRO/react-piwik-pro/issues/23 here, as the relevant code is now in this repo:

While the Piwik PRO service support using [custom data layer names](https://developers.piwik.pro/en/latest/tag_manager/data_layer_name.html), this library seems to have used a [hardcoded default value](https://github.com/PiwikPRO/tracking-base-library/blob/master/src/core/index.ts#L50) `dataLayer`.

I'm wondering whether it would be possible for the library to support initialising the tracker with a custom data layer name. Looking at the code it shouldn't be too complicated to add a new parameter to the initialize function. For example `initialize(containerId: string, containerUrl: string, nonce?: string, dataLayerName?: string).`

**Comment by auto200** [comment]
Hey Axel, thank You for moving the issue. I'm happy to announce that this feature is on our roadmap and will be implemented shortly :smile:

**Comment by auto200** [comment]
@axel-havukangas-tt Customization of Data Layer name has been released in `@piwikpro/tracking-base-library@1.2.0` and `@piwikpro/react-piwik-pro@2.1.0` Now we can provide the name when specifying object as the third argument of `initialize` function

```ts
initialize("container-id", "container-url", { dataLayerName: "my-data-layer-name" })
```
